core/store/src/trie/mem/iter.rs

```diff
@@ -3,13 +3,14 @@
 //! This is essentially a copy of the `DiskTrieIterator`, with the following notable differences:
 //!  - It doesn't support extra options like remembering nodes or puning;
 //!  - It uses None to represent an "empty" placeholder node rather than `TrieNode::Empty`;
-//!  - MemTrieNodeView splits Branch and BranchWithValue into separate variants, whereas TrieNode
+//!  - MemTrieNodeViewGeneric splits Branch and BranchWithValue into separate variants, whereas TrieNode
```

Comment:
We didn't change the name yet?

Reply:
haha, it was renamed to that while I was working on it, but this fell through the cracks when renaming it back :P